fintech-examples/fintech-ui/src/app/payment/initiate/initiate.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'app-initiate',
+  templateUrl: './initiate.component.html',
+  styleUrls: ['./initiate.component.scss']
+})
+export class InitiateComponent implements OnInit {
+  public static ROUTE = 'initiate';
+
+  paymentForm: FormGroup;
+  constructor(private formBuilder: FormBuilder) {}
+
+  ngOnInit() {
+    this.paymentForm = this.formBuilder.group({
+      name: ['', Validators.required],
+      ibanDebitor: ['', Validators.required],
+      ibanCreditor: ['', Validators.required],
+      amount: ['', [Validators.required, Validators.min(0)]],
+      purpose: ['']
+    });
+  }
+
+  onConfirm() {
+    console.log(this.paymentForm.get('pin').value);
+  }
```

Comment:
actually this should go to page with confirmation of details of payment. I dont see page in code and this get pin currently causes error. on confirimation page I will initiate call to BE.

Reply:
Have no api yet and just print the form